Applications/VpView/vpProjectEditor.ui

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>vpProjectEditor</class>
+ <widget class="QDialog" name="vpProjectEditor">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>463</width>
+    <height>131</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Configure - VisGUI View</string>
+  </property>
+  <layout class="QFormLayout" name="formLayout">
+   <item row="0" column="0">
+    <widget class="QLabel" name="datasetLabel">
+     <property name="toolTip">
+      <string>Path to the project imagery. This may be an image list file, or a glob expression of image files.</string>
+     </property>
+     <property name="text">
+      <string>&amp;Imagery</string>
+     </property>
+     <property name="buddy">
+      <cstring>dataset</cstring>
+     </property>
+    </widget>
+   </item>
+   <item row="0" column="1">
+    <layout class="QHBoxLayout" name="datasetLayout">
+     <property name="spacing">
+      <number>2</number>
+     </property>
+     <item>
+      <widget class="QLineEdit" name="dataset">
+       <property name="toolTip">
+        <string>Path to the project imagery. This may be an image list file, or a glob expression of image files.</string>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <widget class="QToolButton" name="pickDataset">
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+         <horstretch>0</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
+       </property>
+       <property name="text">
+        <string>...</string>
+       </property>
+       <property name="icon">
+        <iconset resource="../../Icons/vpView.qrc">
+         <normaloff>:/icons/16x16/browse</normaloff>:/icons/16x16/browse</iconset>
+       </property>
+       <property name="popupMode">
+        <enum>QToolButton::InstantPopup</enum>
+       </property>
+      </widget>
+     </item>
+    </layout>
+   </item>
+   <item row="1" column="0">
+    <widget class="QLabel" name="tracksLabel">
+     <property name="toolTip">
+      <string>Path to tracks file.</string>
+     </property>
+     <property name="text">
+      <string>&amp;Tracks</string>
```

Comment:
@mattdawkins had a request to change this to "tracks or detections". However (besides that being obnoxiously long for a label), we use "tracks" pervasively throughout the UI, and I don't think it makes sense for this one place to be different. Maybe it *does* make sense to revisit our terminology, but if so, I feel strongly that any changes should be made on a more "global" scale.

(OTOH, I have no objection to fiddling with the tool tip; @mattdawkins, would that be sufficient?)

Reply:
'Tracks' has confused people who just work with Detections only. KWIVER also uses both Detections and Track terminology, I would prefer viame/master have both unless it looks really ugly in the GUI.